tests/template_tests/syntax_tests/test_basic.py

```diff
@@ -346,6 +346,61 @@ def test_ignores_strings_that_look_like_format_interpolation(self):
         output = self.engine.render_to_string("tpl-weird-percent")
         self.assertEqual(output, "% %s")
 
+    @setup({"template": "{{ class_var.in_template }} | {{ class_var.render_class }}"})
```

Comment:
Assuming the intent is to use a class method, shall we be very explicit using:
```suggestion
    @setup({"template": "{{ class_var.in_template }} | {{ class_var.class_method }}"})
```

Reply:
With the same spirit, perhaps the following also helps future readers to understand the goal of the test?
```suggestion
    @setup({"template": "{{ class_var.class_instance }} | {{ class_var.class_method }}"})
```